recipes/chardet/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "chardet" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "e53e38b3a4afe6d1132de62b7400a4ac363452dc5dfcf8d88e8e0cce663c68aa" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I just realized that the github repo is way ahead of the release, so the `setup.py` isn't reflecting what's in the tarball. It should be `- chardetect = chardet.chardetect:main`